tests/scripts/expect/_multinode.exp

```diff
@@ -29,7 +29,7 @@
 
 source "tests/scripts/expect/_common.exp"
 
-proc setup_nodes {} {
+proc setup_nodes {{childmode {rs}}} {
```

Comment:
note: bit `s` is being removed in https://github.com/openthread/openthread/pull/5560